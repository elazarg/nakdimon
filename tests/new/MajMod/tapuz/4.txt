שָׁלוֹם, כַּמָּה שְׁאֵלוֹת בְּקֶשֶׁר למגברים כשקונים מֻגְבָּר מנורות בַּיָּד 2 יֵשׁ דְּבָרִים מְיוּחָדִים שֶׁצָּרִיךְ לָשִׂים לֵב אֲלֵיהֶם? מָה שֶׁאֲנִי יוֹדֵעַ זֶה פָּשׁוּט לְנַגֵּן קְצָת וְלִרְאוֹת שֶׁהַכּול עוֹבֵד וְאֵין רעשים לֹא רְצוּיִים. וְהַאִם צָרִיךְ לְהַחֲלִיף מְנוֹרָה כָּל כַּמָּה שָׁנִים? (זֶה מַשֶּׁהוּ שֶׁאֲנִי אוֹכֶל לַעֲשׂוֹת? מָה מְחִירֵי הַמְּנוֹרוֹת?) האקוסטית שֶׁלִּי נִשְׁמַעַת לֹא מַשֶּׁהוּ כְּשֶׁאֲנִי מְחַבֵּר אוֹתָהּ למגבר הָרָגִיל שֶׁלִּי (לֹא מֻגְבָּר לאקוסטית). אַחַת הַבְּעָיוֹת זֶה שֶׁשּׁוֹמְעִים יוֹתֵר מִידַּי אֶת הַיָּד שמחליקה עַל המיתרים. יֵשׁ דֶּרֶךְ לְשַׁפֵּר קְצָת אֶת הַסָּאוּנְד? אֵיךְ בס יִשְׁמַע במגבר לגיטרה חשמלית? כְּמוֹ שאקוסטית נִשְׁמַעַת? אוֹ מַמָּשׁ בִּלְתִּי אֶפְשָׁרִי? (אֲנִי רוֹצֶה לִקְנוֹת בס בִּשְׁבִיל ההתנסות, לֹא רוֹצֶה לְהַשְׁקִיעַ הַרְבֵּה בַּהַתְחָלָה)