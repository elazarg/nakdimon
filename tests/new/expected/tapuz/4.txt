שָׁלוֹם, כַּמָּה שְׁאֵלוֹת בְּקֶשֶׁר לְמַגְבְּרִים
כְּשֶׁקּוֹנִים מַגְבֵּר מְנוֹרוֹת בְּיָד 2 יֵשׁ דְּבָרִים מְיוּחָדִים שֶׁצָּרִיךְ לָשִׂים לֵב אֲלֵיהֶם?
מָה שֶׁאֲנִי יוֹדֵעַ זֶה פָּשׁוּט לְנַגֵּן קְצָת וְלִרְאוֹת שֶׁהַכּוֹל עוֹבֵד וְאֵין רְעָשִׁים לֹא רְצוּיִים.
וְהַאִם צָרִיךְ לְהַחֲלִיף מְנוֹרָה כָּל כַּמָּה שָׁנִים?
(זֶה מַשֶּׁהוּ שֶׁאֲנִי אוּכַל לַעֲשׂוֹת? מָה מְחִירֵי הַמְּנוֹרוֹת?)
הָאָקוּסְטִית שֶׁלִּי נִשְׁמַעַת לֹא מַשֶּׁהוּ כְּשֶׁאֲנִי מְחַבֵּר אוֹתָהּ לַמַּגְבֵּר הָרָגִיל שֶׁלִּי (לֹא מַגְבֵּר לְאָקוּסְטִית).
אַחַת הַבְּעָיוֹת זֶה שֶׁשּׁוֹמְעִים יוֹתֵר מִידַּי אֶת הַיָּד שֶׁמַּחֲלִיקָה עַל הַמֵּיתָרִים.
יֵשׁ דֶּרֶךְ לְשַׁפֵּר קְצָת אֶת הַסָּאוּנְד?
אֵיךְ בָּס יִשָּׁמַע בְּמַגְבֵּר לְגִיטָרָה חַשְׁמַלִּית? כְּמוֹ שֶׁאָקוּסְטִית נִשְׁמַעַת? אוֹ מַמָּשׁ בִּלְתִּי אֶפְשָׁרִי? (אֲנִי רוֹצֶה לִקְנוֹת בָּס בִּשְׁבִיל הַהִתְנַסּוּת, לֹא רוֹצֶה לְהַשְׁקִיעַ הַרְבֵּה בַּהַתְחָלָה)
